src/app/messages/en/attributionText.ts

```diff
@@ -65,12 +65,12 @@ export const defaultText = `
 export const teaText = `
 <p>
   Want to cite, share, or modify this book? This book is
-  <a href="https://creativecommons.org/licenses/by/4.0/">
+  <a target="_blank" rel="noopener" href="https://creativecommons.org/licenses/by/4.0/">
```

Comment:
Should these links also have `nofollow`?
Btw. there is also `addTargetBlankToLinks.ts`, but in this case, I think it's better to hardcode

Reply:
It hasn't been required

Reply:
Since you've added a `noopener` that is not in the scope of the issue I think you could also add `nofollow` if we want it